polaris/tasks/ocean/customizable_viz/viz_horiz_field.py

```diff
@@ -0,0 +1,188 @@
+import cmocean  # noqa: F401
+import numpy as np
+
+from polaris.ocean.model import OceanIOStep
+from polaris.viz import (
+    determine_time_variable,
+    get_viz_defaults,
+    plot_global_mpas_field,
+)
+
+
+class VizHorizField(OceanIOStep):
+    def __init__(self, component, name, indir):
+        super().__init__(component=component, name=name, indir=indir)
+
+    def runtime_setup(self):
+        section = self.config['customizable_viz']
+        self.mesh_file = section.get('mesh_file')
+        self.input_file = section.get('input_file')
+
+    def run(self):  # noqa:C901
+        section_name = 'customizable_viz_horiz_field'
+        section = self.config[section_name]
+
+        # Determine the projection from the config file
+        projection_name = section.get('projection')
+        central_longitude = section.getfloat('central_longitude')
+
+        # Descriptor is none for the first variable and assigned thereafter
+        descriptor = None
+
+        ds_mesh = self.open_model_dataset(
+            self.mesh_file, decode_timedelta=False
+        )
+        min_latitude = section.getfloat('min_latitude')
+        max_latitude = section.getfloat('max_latitude')
+        min_longitude = section.getfloat('min_longitude')
+        max_longitude = section.getfloat('max_longitude')
+        lat_cell = ds_mesh['latCell'] * 180.0 / np.pi
+        lon_cell = ds_mesh['lonCell'] * 180.0 / np.pi
```

Comment:
```suggestion
        lon_cell = ds_mesh['lonCell'] * 180.0 / np.pi
        lon_cell = xr.where(lon_cell < -180., lon_cell + 360., lon_cell)
```
I think we need something like this per https://urldefense.com/v3/__https://github.com/SciTools/cartopy/issues/2590*issuecomment-3461140393__;Iw!!Bt8fGhp8LhKGRg!DWrF60K-Z0ovo5poTT27_rotJs4SWHNj54KJ4EHWlUfvGYndDzjC7QDYvgEAJTEXI33hz_BYAvaI-zE8VS3-pBtj0A$ but this is as yet untested.